utils/parts.stanza

```diff
@@ -1003,7 +1008,13 @@ defn to-jitx (lp: LandPatternCode, jitx-pads-by-pcb-pad-name: HashTable<String,
 
     do(to-jitx, layers(lp))
     do(to-jitx, geometries(lp))
-    do(to-jitx, model3ds(lp))
+    ; TODO: JITX-5895 - Add 3D models for parametric queries
+    ;
+    ;       For now, we don't add 3D models, because we don't know the relative transformations.
+    ;
+    ;       If users want 3D models, they can use "Create component" in component search,
+    ;       which generates Stanza files, that include a comment about the relative transformations.
+    ;do(to-jitx, model3ds(lp))
```

Comment:
@erwin-lau @bhusang Just wanted to highlight this change, which I added since you accepted.

More details on JITX-5895.

Reply:
We should highlight the components that contain 3D models and use that highlighting to hint to the user to use the `Create Component` option to get the 3D data where the user still needs to figure out the appropriate coordinate transform.

Reply:
I am gong to close the PR since the ESIR change will be not necessary